src/huntsman/pocs/states/huntsman/parking.py

```diff
@@ -7,5 +7,11 @@ def on_enter(event_data):
 
     pocs.next_state = 'parked'
 
+    pocs.say("I'm closing the dome.")
```

Comment:
This isn't necessarily true, for instance if the observatory has no dome. This may cause some confusion in the logs.

Reply:
What do you think about the following instead:
```
if pocs.observatory.has_dome:
    pocs.say("I'm opening the dome.")
```